### ISSUE TITLE
Accessibility: Image with no alt attribute - Exhibit Items

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#420]
**Description**
Images in many exhibits do not have alt attributes. This is present in standalone images on a page ([standalone example](https://my2.siteimprove.com/Inspector/602695/Accessibility/Page?pageId=51670201273&impmd=3CC33A985F387DC37EA87D3D5A2306C4#/Criterion/1.1.1/Check/1)) and in image galleries/carousels ([carousel example](https://my2.siteimprove.com/Inspector/602695/Accessibility/Page?pageId=49970042714&impmd=3CC33A985F387DC37EA87D3D5A2306C4#/Criterion/1.1.1/Check/1))

**Solutions**
Option 1: Add a required field for an alt attribute for any image item that is created.
Option 2: Make the Title field required and use that as the alt attribute for all image items.
Option 3: If images have a description/caption, they can be associated with images using the [figure/figcaption HTML5 attribute](https://www.w3.org/WAI/tutorials/images/complex/#structurally-associating-the-image-and-its-adjacent-long-description-html5) or [aria-describedby](https://www.w3.org/WAI/tutorials/images/complex/#description-containing-textual-information)

**Additional Information**
This issue impact all of the exhibits on the site in one way or another. It would be good to fix this issue at the template level and then re-run a scan to identify any that may be solely the responsibility of content owners.
